From bb5560829212c8ffec857fa7b2127f94c17ec89e Mon Sep 17 00:00:00 2001
From: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
Date: Wed, 31 Jul 2013 14:41:25 +0900
Subject: [PATCH 0831/1083] Revert "drm: rcar-du: Add a module argument to
 enable panning in fbdev emulation"

This reverts commit 7f36fbf774c973ef3e0d2581e68c33cc0da76a3e.
---
 drivers/gpu/drm/rcar-du/rcar_du_kms.c | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_kms.c b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
index f54ac20..e9d44b5 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_kms.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
@@ -11,8 +11,6 @@
  * (at your option) any later version.
  */
 
-#include <linux/moduleparam.h>
-
 #include <drm/drmP.h>
 #include <drm/drm_crtc.h>
 #include <drm/drm_crtc_helper.h>
@@ -26,9 +24,6 @@
 #include "rcar_du_lvdsenc.h"
 #include "rcar_du_regs.h"
 
-static bool rcar_du_fbdev_pan = false;
-module_param_named(fb_cma_pan, rcar_du_fbdev_pan, bool, 0444);
-
 /* -----------------------------------------------------------------------------
  * Format helpers
  */
@@ -281,8 +276,7 @@ int rcar_du_modeset_init(struct rcar_du_device *rcdu)
 	drm_helper_disable_unused_functions(dev);
 
 	fbdev = drm_fbdev_cma_init(dev, 32, dev->mode_config.num_crtc,
-				   dev->mode_config.num_connector,
-				   rcar_du_fbdev_pan ? 2 : 1);
+				   dev->mode_config.num_connector, 1);
 	if (IS_ERR(fbdev))
 		return PTR_ERR(fbdev);
 
-- 
1.8.3.2

